dashboard/assets/components/Common.jsx

```diff
@@ -0,0 +1,29 @@
+// isNullOrUndefined returns true if the given variable is null or undefined.
+export const isNullOrUndefined = variable => variable === null || typeof variable === 'undefined';
+
+// defaultZero returns 0 if the given element is null or undefined, otherwise returns the given element.
+export const defaultZero = elem => isNullOrUndefined(elem) ? 0 : elem;
```

Comment:
Unless really really often needed, please just inline the actual code instead of the defaultZero (avoids problems with non-int types).